remove-vowels/io.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "input": ["aaaaa"],
+    "output": [""]
```

Comment:
the output should not be in square brackets

Reply:
@DinnerDinner

Reply:
Changed it, thank you!

Reply:
I didn't make myself, clear: all the outputs should not be in lists, unless the expected output of the function is a list (but judging by the description this is not the case)

Reply:
Ohhh, ok ok, fixed that